doc/ref_arch/kubernetes/chapters/chapter07.md

```diff
@@ -104,7 +104,7 @@ Note: with an underlying IaaS this is possible, but then it introduces (undesira
 
 **Related requirements:** `nfvi.com.cfg.004` and `nfvi.com.cfg.002`
 
-**Baseline project:** _Kubernetes v1.21_
+**Baseline project:** _Kubernetes v1.22_
```

Comment:
@pgoyal01 suggests to remove hardcoded value and replace with reference to table in readme

Reply:
opened #2612 for this